src/components/form/input.tsx

```diff
@@ -2,13 +2,15 @@ import React, { useRef } from 'react'
 import styled, { css } from 'styled-components'
 import { Text } from '../elements'
 import device from 'themes/device'
+import { Localize, localize } from 'components/localization'
+import { TString } from 'types/generics'
 // SVG Component
 import CrossIcon from 'images/svg/help/cross.svg'
 
 interface ReactInput extends React.ComponentPropsWithoutRef<'input'> {
     height?: string
     id?: string
-    label?: string
+    label?: TString | ''
```

Comment:
:thinking:

Reply:
<img width="447" alt="Screenshot 2023-03-09 at 10 50 32 AM" src="https://user-images.githubusercontent.com/125247833/223903768-7c77f5ec-ccd8-4f0a-9a53-c86113ccc388.png">
Its because label got initialize as '' here, that's why i put that type 😄

Reply:
I guess we can remove that :thinking:

Reply:
because it was optional they added an empty string for the type, now we are checking if label exist show that so we can remove empty string from the types